### DISCUSSION TITLE
Is there a way to remove the "stream ended" clip?

### DISCUSSION THREAD

**Discussion by edmael** [owncast/owncast#1959] [Q&A]
Hey, the "stream ended" clip (video + sound) is pretty annoying.
Is there a way to remove that via admin interface or editing some server-side configuration?

Thanks!

**Comment by gabek** [comment]
There is no way to completely remove it.

**Reply by edmael** [reply]
Thanks for the answer! Can I edit something or can you point me to where that clip is stored so I can at least mute the audio?

**Reply by WaiserHai** [reply]
Isn't there a way not to remove it completely? thats mhh...

**Reply by gabek** [reply]
There's a thread that has more detail that might be useful to you but no, you cannot remove it.  https://github.com/owncast/owncast/issues/1965
